build.sbt

```diff
@@ -31,13 +31,13 @@ lazy val root = (project in file("."))
     name := "sbt-bintray",
     sbtPlugin := true,
     libraryDependencies ++= Seq(
-      "org.foundweekends" %% "bintry" % "0.5.0",
+      "org.foundweekends" %% "bintry" % "0.6.0",
       "org.slf4j" % "slf4j-nop" % "1.7.7"), // https://github.com/softprops/bintray-sbt/issues/26
     resolvers += Resolver.sonatypeRepo("releases")
   )
-  .settings(
-    scalaVersion := "2.12.2",
-    sbtVersion in Global := "1.0.0-M5",
-    scalaCompilerBridgeSource :=
-      ("org.scala-sbt" % "compiler-interface" % "0.13.15" % "component").sources
-  )
+//  .settings(
+//    scalaVersion := "2.12.2",
+//    sbtVersion in Global := "1.0.0-M5",
+//    scalaCompilerBridgeSource :=
+//      ("org.scala-sbt" % "compiler-interface" % "0.13.15" % "component").sources
+//  )
```

Comment:
Here is the problem - I am not able to deal with the dependencies here:

```
[info] Resolving org.scala-sbt#launcher;1.0.0 ...
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::          UNRESOLVED DEPENDENCIES         ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: org.scala-sbt#scripted-sbt;1.0.0-M5: not found
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
```

I got an impression that `sbt-scripted` was published somehow differently for `1.0.0`